### PR TITLE
Update info about device credentials

### PR DESCRIPTION
**Pull request by MaratMussabekov**
https://github.com/MicrosoftDocs/windows-itpro-docs/issues/8532

### CODE REVIEW

**Comment by MaratMussabekov** [comment]
Dear @JohanFreelancer9,
kindly do the copy editor review for this PR.

Dear @nam31, for your information

Thank you

**Review by illfated — APPROVED** [review]

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by MaratMussabekov** [comment]
Hello @JohanFreelancer9,
"it is not yet supported for MDM solutions (including Intune) yet"
I used "yet" two times...
which one would you recommend to remove?

Thank you!

**Comment by illfated** [comment]
Just in case this helps you start editing before JohanFreelancer9 replies.
The last "yet" is the one you should remove. As I view it, this makes sense:

> **Device Credential** Credential Type will also work, however, it is not yet supported for MDM solutions (including Intune). It is not recommended to use this option until support is announced.

Reasoning: "not yet" is a very common phrase in English, there is also the case of keeping focus on the importance of "(including Intune)" by removing any less important details after the Intune part.

File: windows/client-management/mdm/enroll-a-windows-10-device-automatically-using-group-policy.md [hunk 1]
```diff
@@ -110,7 +110,10 @@ Requirements:
 
     ![MDM policies](images/autoenrollment-mdm-policies.png)
 
-4. Double-click **Enable automatic MDM enrollment using default Azure AD credentials** (previously called **Auto MDM Enrollment with AAD Token** in Windows 10, version 1709). For ADMX files in Windows 10, version 1903 and later, select **User Credential** (support for Device Credential is coming) as the Selected Credential Type to use. User Credential enrolls Windows 10, version 1709 and later once an Intune licensed user logs into the device. Device Credential will enroll the device and then assign a user later, once support for this is available.
+4. Double-click **Enable automatic MDM enrollment using default Azure AD credentials** (previously called **Auto MDM Enrollment with AAD Token** in Windows 10, version 1709). For ADMX files in Windows 10, version 1903 and later, select **User Credential** as the Selected Credential Type to use. 
+
+> [!NOTE]
+> **Device Credential** Credential Type will also work, however, it is not yet supported for MDM solutions (including Intune) yet. It is not recommended to use this option until support is announced.
```
Comment (JohanFreelancer9) [inline windows/client-management/mdm/enroll-a-windows-10-device-automatically-using-group-policy.md:116]:
```suggestion
> **Device Credential** Credential Type will also work, however, it is not yet supported for MDM solutions (including Intune). We don't recommend using this option until support is announced.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Dear @MaratMussabekov - I agree with @illfated, and have also made a further suggestion.

**Comment by illfated** [comment]
Fair point, good suggestion. I agree with the further suggestion.

**Comment by nam31** [comment]
@Dansimp PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by illfated** [comment]
Looks good. Just give @MaratMussabekov time to merge the last suggestion from @JohanFreelancer9 and it will be good to go.

**Review by JohanFreelancer9 — APPROVED** [review]

**Review by illfated — APPROVED** [review]

**Comment by illfated** [comment]
#sign-off

**Comment by ManikaDhiman** [comment]
Reached out to SME to review the changes.

**Comment by illfated** [comment]
Thank you for this PR, @MaratMussabekov .

**Pull request merged**